params/licenses.go

```diff
@@ -0,0 +1,37 @@
+package params
+
+import (
+	_ "embed"
+
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type DepLicense struct {
+	Name    string  `json:"name"`
+	Version *string `json:"version"`
+	Type    string  `json:"type"`
+	Path    string  `json:"path"`
+}
+
+var (
+	// Polygon Edge License
+	License string
+
+	// Dependency Licenses
+	//go:embed bsd_licenses.json
```

Comment:
Minor nitpick, space between `//` and the rest of the comment should be present

Reply:
This is the correct syntax. It will not be embedded if adding the space.
Please check document: https://pkg.go.dev/embed

Reply:
Really weird syntax 🤦